docs/src/model_creation/examples/basic_CRN_library.md

```diff
@@ -54,12 +54,12 @@ The two-state model describes a component (here called $X$) which can exist in t
 ```@example crn_library_two_states
 using Catalyst, OrdinaryDiffEq, StochasticDiffEq, Plots
 two_state_model = @reaction_network begin
-    (k1,k2), X₁ <--> X₂
+    (k₁,k₂), X₁ <--> X₂
 end
 
 u0 = [:X₁ => 50.0, :X₂ => 50.0]
 tspan = (0.0, 1.0)
-ps = [:k1 => 2.0, :k2 => 3.0]
+ps = [:k₁ => 2.0, :k₂ => 3.0]
```

Comment:
I don't really care about `k₁` and `k1` (the first is pretty, but the second one you can actually write everywhere). I think on this page I think it is nice to use the pretty form in some case (I think it is nice to use different approaches in the examples). In the more of actual docs the more sense it makes to just write `k1`.

Reply:
Here I chaneged it purely for consistency in the model (i.e. we use `X₁`, then `k₁` makes sense as well)